packages/amplify-graphql-generation-transformer/src/__tests__/__snapshots__/amplify-graphql-generation-transformer.test.ts.snap

```diff
@@ -0,0 +1,401 @@
+  const prompt = \\"\\";
```

Comment:
We give so much information and it still returns a response with wrong format? :)

Reply:
Yea 😞 
We haven't found a generic solution yet, but we've seen some promising results with various techniques, and are cautiously optimistic that we can get there.